### PR TITLE
Update mde-device-control-device-installation.md

### PR DESCRIPTION
**Pull request by tewchen**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/mde-device-control-device-installation.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ In Microsoft Endpoint Manager [https://endpoint.microsoft.com/](https://endpoint
     :::image type="content" source="../../media/devicepolicy-devicemgr.png" alt-text="device control":::
 
     > [!NOTE]
-    > Some device in the system have several layers of connectivity to define their installation on the system. USB thumb-drives are such devices. Thus, when looking to either block or allow them on a system, it's important to understand the path of connectivity for each device. There are several generic Device IDs that are commonly used in systems and could provide a good start to build an ‘Allow list’ in such cases. See below for the list:
+    > Some device in the system have several layers of connectivity to define their installation on the system. USB thumb-drives are such devices. Thus, when looking to either block or allow them on a system, it's important to understand the path of connectivity for each device. There are several generic Device IDs that are commonly used in systems and could provide a good start to build an ‘Allow list’ in such cases. Following is one example (not always the same for all USB, you have to understand the PnP tree of the device you want to manage through the Device Manager):
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mde-device-control-device-installation.md]:
```suggestion
    > Some devices in the system have several layers of connectivity to define their installation on the system. USB thumb drives are such devices. Thus, when looking to either block or allow them on a system, it's important to understand the path of connectivity for each device. There are several generic device IDs that are commonly used in systems and could provide a good start to build an ‘Allow list’ in such cases. The following is one example (it is not always the same for all USBs; you need to understand the PnP tree of the device you want to manage through the Device Manager):
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tewchen, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you @tewchen @JohanFreelancer9

**Pull request merged**